### DISCUSSION TITLE
internal and external certs: 2 listeners?

### DISCUSSION THREAD

**Discussion by AxisNL** [drakkan/sftpgo#840] [General]
Hey I want to use sftpgo as a file exchange server to exchange stuff between the inside of my network and the outside of my network.

Internally I have a really locked down infrastructure with my own certificates (ftp.internal.local), and on the outside i have servers with public certs (ftp.company.com). I don't want to do split DNS. What we normally do with multiple applications is have our servers in the DMZ have two ip's, where one ip is bound to a public cert, and one ip is bound to an internal cert.

But looking at the sftpgo configuration file example, I't doesnt seem like I can add different certs to individual bindings?

How would you solve this scenario? Another option might be 2 sftpgo servers with a shared database (can these be shared?), and run different certs on different servers?

Angelo.

**Comment by drakkan** [comment]
> Hey I want to use sftpgo as a file exchange server to exchange stuff between the inside of my network and the outside of my network.
> 
> Internally I have a really locked down infrastructure with my own certificates (ftp.internal.local), and on the outside i have servers with public certs (ftp.company.com). I don't want to do split DNS. What we normally do with multiple applications is have our servers in the DMZ have two ip's, where one ip is bound to a public cert, and one ip is bound to an internal cert.
> 
> But looking at the sftpgo configuration file example, I't doesnt seem like I can add different certs to individual bindings?
> 

I think this is not possible I'm sorry, maybe I can add this feature in the future

> How would you solve this scenario? Another option might be 2 sftpgo servers with a shared database (can these be shared?), and run different certs on different servers?
> 

MySQL, Postgres and CrDB can be shared, this is supported, but honestly it seems overkill for this use case

> Angelo.
